docs/sources/getting-started/glossary.md

```diff
@@ -40,12 +40,30 @@ This topic lists words and abbreviations that are commonly used in the Grafana d
       Explore allows a user to focus on a building a query. User can refine the query to return the expected metrics before building a dashboard.
     </td>
   </tr>
+  <tr>
+    <td style="vertical-align: top">Exporter</td>
```

Comment:
If you add exporter, consider adding "export" as well. I'm concerned our readers will confuse this with the import/export dashboards functionality.

Reply:
Will do